.github/workflows/acceptance_tests.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         with:
           ruby-version: '2.6'
 
-      - name: Fix common Linux and MacOs permissions
+      - name: Fix common Linux and macOS permissions
```

Comment:
![little mac](https://static.wikia.nocookie.net/character-stats-and-profiles/images/f/fc/Little-mac-punch-out-little-mac-sprite-11563134844z5gam4ptzb.png)